spacy/lang/sr/tag_map.py

```diff
@@ -0,0 +1,27 @@
+# coding: utf8
```

Comment:
This is the default tag map, right? In that case, we could probably also leave it out for now, because it's what spaCy should default to anyways.

Reply:
Yes, you're right. I've removed the file and updated PR.